packages/legend-query/src/stores/QueryBuilderLambdaProcessor.ts

```diff
@@ -971,6 +987,10 @@ export class QueryBuilderLambdaProcessor
       let currentPropertyExpression: ValueSpecification = valueSpecification;
       while (currentPropertyExpression instanceof AbstractPropertyExpression) {
         const propertyExpression = currentPropertyExpression;
+        isValidMilestoningLambda(
```

Comment:
I think the naming here is confusing. It gives me the impression that this is a checker, but looking at it, it returns `void` so I know the function is not pure, i.e. it modifies something, so I have to jump to that bit of the code and check. Would be great if we name these kinds of function more accurately (preferably with inline comment and function comment) to lessen the number of jump around we have to make while reading this code